massa-execution/src/vm.rs

```diff
@@ -194,10 +195,14 @@ impl VM {
             for (op_idx, operation) in block.operations.clone().into_iter().enumerate() {
                 let operation_sc = OperationSC::try_from(operation.content);
                 if operation_sc.is_err() {
-                    // only fail if the operation cannot parse the sender address
+                    // fails if the operation cannot parse the sender address or if the bytecode is invalid
                     continue;
                 }
                 let operation = &operation_sc.unwrap();
+
+                // Prepare context and save the Initial ledger changes before execution
+                // The returned snapshot contains a copy of the initial coin credits
+                // that will be popped back if bytecode execution fails in order to cancel its effects
```

Comment:
I don't think you need to add `in order to cancel its effects`. That makes it read like popping the coins back will have some other side-effect(does it?).

Reply:
done